src/test/java/com/skynest/EditUserByIdTest.java

```diff
@@ -30,7 +38,40 @@ void valid_edit_user_by_id_test(EditRequest editRequest) throws IOException {
 
     @DataProvider(name = "UserData")
     public Object[][] getUserData() {
-        return new Object[][]{
-                new Object[]{EditRequest.generateValidEditRequest()}};
+        return new Object[][]{new Object[]{EditRequest.generateValidEditRequest()}};
+    }
+
+    @Test(dataProvider = "UserData")
+    void edit_random_worker_details_as_admin_test(EditRequest editRequest) throws IOException {
+        loginAs(Roles.ADMIN);
+        Response getAllUsersResponse = skyNestBackendClient.getAllUsers();
+        getAllUsersResponse.then().statusCode(SC_OK);
+
+        List<UserResponse> userResponses = JsonTransformer.mapResponseToList(getAllUsersResponse, UserResponse.class);
+        UUID firstWorkerId = getRandomWorkerId(userResponses);
+
+
+        Response editUserResponse = skyNestBackendClient.editUserById(editRequest, firstWorkerId);
+        editUserResponse.then().statusCode(SC_OK);
+
+        EditResponse editResponse = editUserResponse.as(EditResponse.class);
+
+        Assert.assertNotNull(editResponse.getId());
+
+        Assert.assertEquals(editResponse.getName(), editRequest.getName());
```

Comment:
just a note here for the sake of readability:

you can static import asserEquals and avoid using Assert.assertEquals call to this method...

just a personal preference tbh :)

Reply:
Seems more readable definitely;
Will add junit dependency and will use assertEquals :)

Reply:
No need to use junit's version of assert equals: you could just declare the existing import this way:
`import static org.testng.Assert.assertEquals;`

which would enable you to write the asserts like this:
`assertEquals(editResponse.getName(), editRequest.getName());`